test/gerkin/tests/features/data_my_personal_data_usage.feature

```diff
@@ -0,0 +1,21 @@
+Feature: API endopoint allowing users to check who had read their personal data in the Digital Registries database.
+  Request endpoint: GET /data/MyPersonalDataUsage/1.0/
+
+  Background:
+    Given The user wants to check who had read his personal data in the Digital Registries database
+
+  Scenario: The user receives a list with all records that had read his personal data in the Digital Registries database
+    When The user triggers an action to receive records from the database
+    And The request with a valid payload is sent
```

Comment:
Not enough information, what's the difference when list of records is empty vs full?

Add:
`And There are records that read his personal data in the database`
or
`And There are no records that read user's personal data`